mongo/bounties/final.json

```diff
@@ -0,0 +1,86 @@
+[
+  {
+    "_id": "60f6585b453e70eed340e8e3",
+    "season": 1,
+    "title": "Make new website landing page",
+    "description": "Description for new website landing page with specs",
+    "criteria": "A completed bounty has the following deliverables...",
+    "reward": {
+      "currency": "BANK",
+      "amount": 10000.00
+    },
+    "applicableGuilds": [
+      {
+        "guildName": "Developers Guild",
```

Comment:
This is a minor thing that I won't hold back in the review, but can we omit the word "Guild" in the guild name? It will make displaying it on the front end a lot easier.

Reply:
hmm so the name comes directly from the discord ROLE values which is currently 

name: Developer's Guild
id: 857838853326241814

What I can do is create an issue in DEGEN to have the name converted from w/e discord given to the defined values for the front end. What I need from you guys is a list of guild names to do that mapping. If this is needed for MVP to make things simple let me know and I can get the process started

Reply:
@amaredeus Don't think this is needed for MVP since it is purely cosmetic. I can just as easily strip out the word guild for the tag for now. We can revisit it.